SimTracker/SiPhase2Digitizer/plugins/PixelBrickedDigitizerAlgorithm.cc

```diff
@@ -50,11 +50,12 @@ PixelBrickedDigitizerAlgorithm::PixelBrickedDigitizerAlgorithm(const edm::Parame
 PixelBrickedDigitizerAlgorithm::~PixelBrickedDigitizerAlgorithm() {
   LogDebug("PixelBrickedDigitizerAlgorithm") << "Algorithm deleted";
 }
-void PixelBrickedDigitizerAlgorithm::induce_signal(const PSimHit& hit,
-                                                   const size_t hitIndex,
-                                                   const uint32_t tofBin,
-                                                   const Phase2TrackerGeomDetUnit* pixdet,
-                                                   const std::vector<digitizerUtility::SignalPoint>& collection_points) {
+void PixelBrickedDigitizerAlgorithm::induce_signalBricked(
```

Comment:
why is this changed of name? I thought this was meant to override the method of the mother class:

https://github.com/cms-sw/cmssw/blob/44c3f64c0f5ce5c08f31fb622bc94de026c84263/SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.h#L193-L197

in this way this method is never called.